doc/smart-switch/BFD/SmartSwitchDpuLivenessUsingBfd.md

```diff
@@ -192,18 +194,23 @@ When HA manager receives HA config update, it will create VNET Route tunnel tabl
 
 <img src="images/DB_And_Notification.png">
 
-## 2.7 NPU-DPU midplane state change and BFD session update
+## 2.7 DPU BFD session state Updates
+HA Manager maintains an external facing state table in NPU to publish all the required state information per VDPU that can be queried by the controller. To provide state of BFD sessions established between the local DPU and NPUs, DPU updates the DASH_HA_SCOPE_STATE table with all the BFD sessions that are up.
+

```

Comment:
we need to use a dedicated table for sending the information back to hamgrd, because BFD doesn't know the HA configurations with our workflow.

Reply:
New state_db table is added to get the list of BFD sessions in UP state.